libskale/broadcaster.cpp

```diff
@@ -90,7 +90,8 @@ void* ZmqBroadcaster::server_socket() const {
     if ( !m_zmq_server_socket ) {
         m_zmq_server_socket = zmq_socket( m_zmq_context, ZMQ_PUB );
 
-        int val = 16;
+        // it was 16 before, this caused messages lost during performance tests
+        int val = 1024;
```

Comment:
we should measure memory consumption for this particular change

Reply:
In my tests memory consumption is pretty good, this change does not increase it much.

Normally ZeroMQ is pretty fast broadcasting, so the queue does not grow to large size, 1024 is the worst case that could happen in case of temporary network slowdown. 

We need to wait until the second pull request to run the tests and measure memory, since we split the original test in half. In general memory consumption improved a lot.